redaxo/src/addons/structure/plugins/content/lib/var_template.php

```diff
@@ -15,6 +15,12 @@ protected function getOutput()
             return self::class . '::getTemplateOutput(require ' . self::class . '::getTemplateStream(' . $template_id . ', $this))';
         }
 
+        $template_key = $this->getParsedArg('key', null, true);
+
+        if ('' !== $template_key) {
+            return self::class . '::getTemplateOutput(require ' . self::class . '::getTemplateStream(' . $template_key . ', $this))';
```

Comment:
Die Auflösung des Keys zur ID sollte hier passieren, damit im Cache die ID landet, und nicht bei jedem Aufruf der key zur ID aufgelöst werden muss.

Somit könnte man oben die Methode in rex_template ggf. auch erstmal ganz weglassen, und einfach nur hier direkt aus der DB die ID auslesen.